### DISCUSSION TITLE
Problem in tmrt calculation due to D and N

### DISCUSSION THREAD

**Discussion by difruscj** [UMEP-dev/UMEP#504] [Q&A]
Hello everyone!

I have a problem in tmrt calculations.
I calculated the tmrt with the UMEP plugin on QGIS and I find strange results. During the day there is a discontinuity in the tmrt results which are influenced by the D and N (which I imagine to be Day and Night). I have no control on that, the N influences the values from midnight to 1 pm and the D influences the values from 2 pm to 11 pm. There is a discontinuity in the results because the tmrt in period N is much lower than period D. This discontinuity can go up to a difference of 10°C, which makes a lot of difference in just 1 hour. I don't understand how D and N vary my tmrt results. Can you help me to control the D and N of my tmrt rasters?

Thank you, any help would be helpful!

**Comment by biglimp** [comment]
This sounds like a timing issue somewhere between: 

1. the UTC settings, 
2. timezone in the meteorological file 
3. and/or the lat, lon as well as CRS of your DSM

Have a look and see if you have them correct. Remember that UTC setting should be set according to the timezone in the meteorological file.

**Comment by difruscj** [comment]
Thank you for your help!
